recipes/arviz/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "arviz" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 40cddad16c09a4944071b62c3d2fd0ba8a318bde625af5a4988de8ae085ab3fe
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - matplotlib >=3.0
+    - numpy
+    - scipy
+    - pandas
+    - xarray
+    - netcdf4
+
+test:
+  imports:
+    - arviz
+
+about:
+  home: https://github.com/arviz-devs/arviz
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
```

Comment:
Can you drop these comments from the example

Reply:
Done!